model/src/main/proto/spine_examples/shareaware/identifiers.proto

```diff
@@ -44,3 +46,8 @@ message WatchlistId {
 message ShareId {
     string uuid = 1 [(required) = true];
 }
+
+// Identifies a wallet in the scope of the application.
```

Comment:
Let's not repeat this "in the scope of the application". It's obvious enough.

I would remove it from all descriptions.